tests-ui/tests/nodeSearchService.test.ts

```diff
@@ -6,7 +6,7 @@ const EXAMPLE_NODE_DEFS: ComfyNodeDefImpl[] = [
   {
     input: {
       required: {
-        ckpt_name: [['model1.safetensors', 'model2.ckpt']]
+        ckpt_name: [['model1.safetensors', 'model2.ckpt'], {}]
```

Comment:
We have zod.transform to ensure that the second element always exists. https://github.com/huchenlei/ComfyUI_frontend/blob/e179f75387173a182e7007f0cdb91a2dfaab6d84/src/types/apiTypes.ts#L161-L176